features/user_can_receive_mail.feature

```diff
@@ -0,0 +1,28 @@
+Feature: User is able to receive and reply to email
+    As a User
```

Comment:
formatting

Reply:
Locally the formatting issue does not display on my pc - can you please pull on yours to rectify?